web/l10n/en.json

```diff
@@ -144,6 +144,8 @@
     "commandData": "Data",
     "commandPhone": "Phone Number",
     "commandMessage": "Message",
+    "commandTimezoneOffset": "Timezone Offset to UTC",
```

Comment:
Command key should match the command key for consistency. Also, I think we can shorten the value to "Timezone Offset".